.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,49 @@
+name: Lint and Test Charts
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_call:
+  workflow_dispatch:
```

Comment:
I dont think we need these triggers

Reply:
Added these in case we want to call it manually/remotely (test post-merge/sanity checks)